docs/developer-docs/multi-chain/ethereum/evm-rpc/overview.mdx

```diff
@@ -0,0 +1,85 @@
+---
+keywords: [advanced, ethereum, evm rpc, evm, tutorial]
+---
+
+import { MarkdownChipRow } from "/src/components/Chip/MarkdownChipRow";
+
+# Overview
+
+<MarkdownChipRow labels={["Advanced", "Ethereum", "Tutorial"]} />
+
+**EVM RPC** is an ICP canister smart contract for communicating with [Ethereum](https://ethereum.org/en/) and other [EVM blockchains](https://chainlist.org/?testnets=true) using an on-chain API. Communication with EVM-compatible chains includes obtaining and sending information such as:
+
+- The logs of a specific block or transaction.
+
+- Information about a block.
+
+- Details about a transaction.
+
+- Historical data regarding the network's gas fee.
+
+- Transactions for a specific address.
+
+- Submitting a signed transaction.
+
+The source code for this project is available on GitHub ([internet-computer-protocol/evm-rpc-canister](https://github.com/internet-computer-protocol/evm-rpc-canister) ⭐️) under the Apache 2.0 license.
+
+## Goal
+
+The goal of the EVM RPC canister is to act as a gateway between canisters on ICP and EVM-compatible networks through RPC services such as Alchemy, Ankr, or Cloudflare. The EVM RPC canister provides the necessary methods for other ICP canisters to send requests to the Ethereum JSON-RPC API and receive responses.
+
+This canister is available for all dapp developers to use and adopts a decentralized governance model through the NNS.
```

Comment:
what does it mean that it "adopts a decentralized governance model through the nns"?
a) i think the canister is not yet under NNS control
b) we should link to a resource hear that explains what the NNS controlled canisters are

Reply:
Maybe we could introduce this in a follow-up PR after the canister is moved to the NNS (working on this right now, but it might take a while for the proposal to go through).

Reply:
Removed the info about the NNS for now until the canister is moved to the NNS